mira/dkg/api.py

```diff
@@ -110,4 +207,5 @@ def get_relations(relation_query: RelationQuery, request: Request):
             (dict(s), dict(p) if isinstance(p, Relationship) else [dict(r) for r in p], dict(o))
             for s, p, o in records
         ]
-    return records
+    else:
+        return [RelationResponse(subject=s, predicate=p, object=o) for s, p, o in records]
```

Comment:
@cthoyt I might have missed a bug here. In the `if` branch, the `records` variable is constructed but never used. In the `else` branch, a value is returned.

Reply:
Oops yes this was one of the tricky parts I wasn’t sure about how to type annotate for fast api. Maybe better to make two different endpoints

Reply:
Will send a fix in a bit